OpenOversight/app/templates/tag.html

```diff
@@ -32,9 +32,10 @@ <h1>Tag {{ face.id }} Detail</h1>
                    data-top="{{ face.face_position_y }}"
                    data-width="{{ face.face_width }}"
                    data-height="{{ face.face_height }}"
-                   src="{{ path }}" />
+                   src="{{ path }}"
+                   alt="{{ face.officer.full_name() | title }}" />
             {% else %}
-              <img id="face-img" src="{{ path }}" />
+              <img id="face-img" src="{{ path }}" alt="" />
```

Comment:
`{{ face.officer.full_name() | title }}'s profile image.` feels like a good thing to put here.